pkg/operator/baremetal_config.go

```diff
@@ -37,8 +38,12 @@ type BaremetalProvisioningConfig struct {
 func getBaremetalProvisioningConfig(dc dynamic.Interface, configName string) (BaremetalProvisioningConfig, error) {
 	provisioningClient := dc.Resource(provisioningGVR)
 	provisioningConfig, err := provisioningClient.Get(context.Background(), configName, metav1.GetOptions{})
+	if apierrors.IsNotFound(err) {
+		glog.Infof("Baremetal provisioning CR %s is not found", configName)
+		return BaremetalProvisioningConfig{}, nil
+	}
 	if err != nil {
-		glog.Errorf("Error getting config from Baremetal provisioning CR %s", configName)
+		glog.Infof("Baremetal config not available from Baremetal provisioning CR %s", configName)
```

Comment:
this should log Errorf

Reply:
We are returning error as nil for this condition, so thought that Infof was better than Errorf. Will change it back.

Reply:
Done.